lib/connection/connection_config.js

```diff
@@ -271,6 +271,15 @@ function ConnectionConfig(options, validateCredentials, qaMode, clientInfo)
     authenticator = authenticator.toUpperCase();
   }
 
+  let browserActionTimeout = options.browserActionTimeout;
+  if (Util.exists(options.browserActionTimeout))
+  {
```

Comment:
let's format the code according to eslint rules

Reply:
sure